image_data_guide/04b_tensorflow_training.ipynb

```diff
@@ -0,0 +1,889 @@
gVinzyosv0Ol0zI1SLRvlyT3H6fZ6nD37mtGyC2i2GozGY6I4QdcsyqUqg+GQMDQTgO/6NBomo2nfvn288sorPHLqFI1Gg5dfeYX7T97HTr9HmsW40qXkeVy7epFGs0qn06E7GHLt8hXe/UPvIo5jJsMR0+mYarVKGEWMBkOEtFheXiSNItI0IY4FvueSpSnrN65gux6lWj3Pd9q13vd8h2azSZglBFHIJJjmKh2beDql3WxR8nzq9TpxHBMEAY1yE69cYjIZIXO+zs7W9vc8+X9d4+c+8nP8zu/8P1y9usnl9U0cH049+jBCN7hw4Tx/+ZcvgoZ3v/u9SLuM74GKE6r1GvV6k0984pO8fuaCaRs6Nj/zMz+D1A5ax+xsbCFdB5SmVilx8dxrLN5zjEQIhC0plUoMxxOiKDJmhM0a3b5p0bl+id5OD41FFCWEYcjS0hLTeEoUT3nkbe/g05/5FP3hgCRJsCwbzxZcvXiBi2fO8drpM/S72+w/uEan05n1u8fTKY5ryLO97T6+X+LA/oNIy2I0GBGFEeVSiVKlRJyM2djcJFOCL33xq1x4fZ2f/9lforezxXA8QAmLRrlKEMZopahUakjLIUoTbMdlFE6xXEkqUn77X/4vaK144ol38KEPfQhfZWxs36JWq2HZEp0pfvVXf5Xf/K3fQClNFGc888UvEwQhdk5mbjRqdDeNaWCqY5rNt0a70kyyEgt5G5CQ3D65vlkLZJ4Xc1tydsGH+S5gZv79iw/jvWNvlWPX4XX397MvKbFsG5TCsm0s20Y6jpH550ngxTYVVYMCEOxWhL4LcDMPbtt3kT9O56Tt3227d8GMmY7mQc88KCv6LAY07f4831pK0/QtSQCGXeBrCKxGNuwIC6kFrpWnqwsTGFs0PAqxhVVMyDo3BczJtFoYg7ldku3tAGbGe7HEDGzKQmgtMAwbYYAJOsOiiLzYbb/YwpoBBwuBk/NlUpVhY1FxfGRmWkgBCp2Za9bSILVCoihbLo50SZNwTiK9mwIucyBjWkSaTCekZGhhG18aYZl0bGHNqoiza0rnZHSdYiEo+x6+6yLGKalOZ89XuV/OXhuBvcfqP2S8KZiZTkLq9Tpaa7rdbUZxhOh08Mslbm5s0l5YYHFlH5krSRPBqD+i3Gly6OgaB47sZzqeUK9U+eSnPoFlWTz9iT+DLOOekw9w6OABHjv1CC9+6xukacryQp2vPPMXaGL27z9IEsX45RKrnSavX7yELVJqZY9G2eXKtatkWcD+/fs5e+4sd584zupymzBWeJ7PeGDStB88+RCTyYQ0SukOt2lVq2RZRqfZ4erVq6g0o1IpsbCwyPbODgcOHGB1dZUr169h20Y2XqvVkKmm0Wxy48YNGs0WW7c2eOjRR+hubXP2/Fn63S5Hjxzj2PFjvPjit7n7+F0EYcxwOGRlaYEgCBgNBqSJYmFhwUjGb65juy4Li23C0FR4PM+j2++j2MBxPRKtENLi0oUL1MtVw/8QlvG5aRgTQOnaWJaJnjh89Ahnz55lYalJNar9B18Mf1Xj6pUL/PxH/wZ/8Ief5JG33cNgOubatWtceOU0jUaFD3zgKV6/cInnv/1NlEzRwiMKYlRm8Wef+BQvffsstiWI3gMrAAAgAElEQVQpYbG8bx9Romi3lwiiLqXqIaI4JYlSBqOAwWBiVttCEAdTYzboGgVeGCeoLKFWLzOdmjBIW3oEQYS0TavPcRx0kvAXn/00SZiRZCYMTqcZO9tbxNMJa2trbPe3WVleplVv0Wy2iTOTwByFJok7TVN2dnZYaC/i+yVUqnGkzWQymSWvF5NWuVymVKlSqzf55Kc/y+f+4svGHkDAZBqztOySqgxheWgUlu/jVqvsjLdw/Bq+79Lv9yHLcFybL3/1K3z92W8gpcOv/Xf/mGajRnuhYyp6i4sc2H+I9fV1wnDCaDzJpaEZjWqd8XhImsWGbO7WSFLne5/gv8ZhJhFrDqAYQu+btZrmCbtvqGgojc6MaskkVxflbUWaZcY/ZH6y31Min03u3+Vv3/a3YMbj0lgoBY6Tp95bGmGIEWTZriqokEDPVrF32M/i/Y1bKmaf5qokhdFfUZmZD+As3ltKCfMSdK2QOdl4noysC6WJys308qoX7MYdzIPEtxyYKVBGPmYAUxljN1c4szbdPPARiNyrRYMoLOl2DfEEuXQ7fx7shgw4AmxtJludV4BTwBM2vi1JVEKSpcS5ekyiTXAuYBxb9lRP0GSiiEow1+086d2Qvw1R10KihYXQGTbgIChJSalSIhgmZErtumpjtt9GI8mILFAyw840rsqVTsqEQM7aQDrXPOXXZaqLNHHDMgrDkDSODZBBmyqW2bg3APP5c7K3OvPdrvv58eameZZksWkkv/s6i1y/fpXtjU0cz86N3HYtrO1qGR1HZJni+qiL7Vlo4UPJ44kffh9HDq2RJMbHolr2GA9HNOsHqHg2+/fvZzSacN9991KpVDl/+jssLnS4cn4Ly7GZRiF//sd/RMnzefn5Lls720RhgnQd2u02N69fod1uU691qDfNCtlzS3jSZpykJHGMZzvsW1wiCCJcx6fV7JDk5D+hLRYXOkymY3r9LpevXaXerLG8vIywLEaTAaNpwPFjR7i5fot6vW7089Jhod1m2O/j+Q4vvvgi42BKaTTE9T1atQbDUR9LGIfWQa+P60ik7dOu16g0GwzGI6IgoFwuI6WcxRq8/voFnJJHEARIKUmjGAvBTq9Lo9VkNBpRqVSwHEmWpDQaDXZ2dlg9cIAsiFlZeevEGWxsXkfolP/sJ9+FUypz9cY6t4IIv1RhMpkwnU6Iwj6HDnfwPI+Vpf3U2x1eee11xkFqHHOHEbYjuLW5w+/94b/Bti1++b/6W0zDmChJOHvuElcuXmE87tIfjilVypTLEt8zl/h0OsGr1FBRymQyoeTX0Vk4K/UXdu3VSokwUgz6Yz719NNU63UmkwnDyQjPltQrVSbDEQKoV6s
```

Comment:
Optional [i.e. I could add this in myself]. A small blurb to explaining what notebooks come next. 

e.g. "You can also explore how to train a model using TensorFlow or PyTorch, see...."

Reply:
Added link to deploying models in sagemaker developer guide.